findbugs-filter.xml

```diff
@@ -3,5 +3,8 @@
     <Match>
         <Class name="~com\.microservicesteam\.adele\..+\.domain\.Immutable.*" />
     </Match>
+    <Match>
+        <Class name="~com\.microservicesteam\.adele\..+\.model\.Immutable.*" />
```

Comment:
@kelemena this raises another question: shouldn't the model classes be in the `domain` package in `adele-ticket-master-api` module?